test/mechanics/cell/SimpleAssembler.cpp

```diff
@@ -101,17 +100,17 @@ BOOST_AUTO_TEST_CASE(AssemblerHessian)
      *     active J         dependent K
      */
 
-    NuTo::GlobalDofMatrixSparse hessian =
+    NuTo::DofMatrixSparse<double> hessian =
             assembler.BuildMatrix({mockCell0.get(), mockCell1.get()}, {d}, NuTo::CellInterface::MatrixFunction());
-    Eigen::Matrix3d JJ = (Eigen::Matrix3d() << 11, 12, 13, 21, 22, 23, 31, 32, 44).finished();
-    Eigen::Matrix2d KK = (Eigen::Matrix2d() << 22, 23, 32, 33).finished();
-    Eigen::MatrixXd JK = (Eigen::MatrixXd(3, 2) << 0, 0, 0, 0, 12, 13).finished();
-    Eigen::MatrixXd KJ = (Eigen::MatrixXd(2, 3) << 0, 0, 21, 0, 0, 31).finished();
-
-    BoostUnitTest::CheckEigenMatrix(Eigen::MatrixXd(hessian.JJ(d, d)), JJ);
-    BoostUnitTest::CheckEigenMatrix(Eigen::MatrixXd(hessian.JK(d, d)), JK);
-    BoostUnitTest::CheckEigenMatrix(Eigen::MatrixXd(hessian.KJ(d, d)), KJ);
-    BoostUnitTest::CheckEigenMatrix(Eigen::MatrixXd(hessian.KK(d, d)), KK);
+
+    Eigen::MatrixXd hessianE = (Eigen::MatrixXd(5,5) << 11, 12, 13,  0,  0,
+                                                       21, 22, 23,  0,  0,
+                                                       31, 32, 44, 12, 13,
+                                                        0,  0, 21, 22, 23,
```

Comment:
That is much clearer now! :+1: 
I feared that clang format would destroy that precious formatting...

Reply:
You can always use `// clang-format off` and `// clang-format on` to forbid clang format to destroy your precious formatting ... my precious :trollface: